pkg/services/ngalert/ngalert.go

```diff
@@ -94,9 +94,9 @@ func (ng *AlertNG) init() error {
 
 	baseInterval := ng.Cfg.AlertingBaseInterval
 	if baseInterval <= 0 {
-		baseInterval = defaultBaseIntervalSeconds
+		baseInterval = defaultBaseIntervalSeconds * time.Second
+		ng.Cfg.AlertingBaseInterval = baseInterval
```

Comment:
nit, `AlertingBaseInterval` should better move under `Cfg.UnifiedAlerting`.
I have noticed also that we never `ng.Cfg.AlertingBaseInterval` so it always gets the default one.

Reply:
Yes. this is a hack until https://github.com/grafana/grafana/pull/45107 is  merged. I would also appreciate your review of it.